cis/system/ksp-nist-cm-11-1-allow-user-installation-of-software-only-with-explicit-privileged-status.yaml

```diff
@@ -0,0 +1,25 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit: 
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorHostPolicy
+metadata:
+  name: ksp-nist-cm-11-1-allow-user-installation-of-software-only-with-explicit-privileged-status
```

Comment:
if the policy is for VM branch then please use `vm-hsp-<name-for-policy>` as the name @AbhinavCSY